src/main/java/frc/robot/RobotContainer.java

```diff
@@ -187,23 +185,28 @@ private void configureBindings() {
                 }));
```

Comment:
It's not letting me comment above this, but perhaps turning these to onTrue() simplifies setting the reef level? Also, if you implement the generic arm position enum below, then you can set an arm position variable instead with the proper enum.